website/static/js/notificationsTreebeard.js

```diff
@@ -118,11 +123,30 @@ function ProjectNotifications(data) {
                         filter : true,
                         sortInclude : false,
                         custom : function() {
-                            return m('div[style="padding-left:5px"]',
-                                    [m('p',
-                                        [m('b', item.data.node.title + ':')]
-                                )]
-                            );
+                            var globalNotificationsMessage = 'These are default settings for new ' +
+                                'projects you create or are added to. Modifying these settings will ' +
+                                'not modify settings on existing projects.';
+                            var projectNotificationsMessage = 'These are settings for each of your ' +
+                                'projects. Modifying these settings will only modify the settings ' +
+                                'for the selected project.';
+                            return m('div[style="padding-left:5px; padding-bottom:50px"]', [
+                                m('p', [
+                                    m('b', item.data.node.title + ':  '),
+                                    item.data.node.title === 'Default Global Notification Settings' ?
+                                        m('span[class="fa fa-info-circle"]', {
+                                            'data-toggle': 'tooltip',
+                                            'title':globalNotificationsMessage,
+                                            'config': popOver,
+                                            'data-placement': 'bottom'
+                                        }) :
+                                        m('span[class="fa fa-info-circle"]', {
```

Comment:
Or, even better: add the help text to this function: https://github.com/CenterForOpenScience/osf.io/blob/6e449d62a128163a6e9519a5d51ca2ed28ccf503/website/notifications/utils.py#L403 and use it here.

```python
    return [
        {
            'node': {
                'id': user._id,
                'title': 'Default Global Notification Settings',
                'help': 'These are the default settings...'
            },
        },
```